frontend/src/app/testQueue/AoEForm/AoEModalForm.jsx

```diff
@@ -321,28 +322,38 @@ const AoEModalForm = ({
     onClose();
   };
 
-  const actionButtons = (
-    <div style={{ float: "right" }}>
-      <Anchor text="Cancel" onClick={onClose} />
+  const buttonGroup = (
+    <div className="sr-time-of-test-buttons" style={{ float: "right" }}>
+      <Button unstyled label="Cancel" onClick={onClose} />
       <Button label={saveButtonText} onClick={saveAnswers} />
     </div>
   );
 
   return (
-    <CMSDialog
-      onExit={onClose}
-      closeText="Cancel"
-      heading={displayFullName(
-        patient.firstName,
-        patient.middleName,
-        patient.lastName
-      )}
-      getApplicationNode={() => {
-        document.getElementById("#root");
+    <Modal
+      isOpen={true}
+      style={{
+        content: {
+          inset: "3em auto auto auto",
```

Comment:
For my information: what determines which style information should be inlined versus going into Sass?

Reply:
I'm not sure if I can make this style a className. The react-modal component says to put these inline but it's a bit of a pain to do that. If we change to the Trussworks modal it may work differently.